apple/RNGestureHandlerManager.mm

```diff
@@ -71,6 +71,14 @@ - (instancetype)initWithUIManager:(RCTUIManager *)uiManager eventDispatcher:(RCT
 
 - (void)createGestureHandler:(NSString *)handlerName tag:(NSNumber *)handlerTag config:(NSDictionary *)config
 {
+  if ([_registry handlerWithTag:handlerTag] != nullptr) {
+    NSString *errorMessage = [NSString
+        stringWithFormat:
+            @"Handler with tag %@ already exists. Please ensure that no GestureHandler instance is used across multiple GestureDetectors.",
```

Comment:
```suggestion
            @"Handler with tag %@ already exists. Please ensure that no Gesture instance is used across multiple GestureDetectors.",
```

Reply:
Done in [0b178e3](https://github.com/software-mansion/react-native-gesture-handler/pull/2694/commits/0b178e350481ae074caccd46657da242d2f48f65)